### DISCUSSION TITLE
Can't use "Custom JS Functions" as input "Format Prompt Values" for Workers

### DISCUSSION THREAD

**Discussion by allankz** [FlowiseAI/Flowise#3450] [Q&A]
To facilitate the publication of variables among workers, I tried using "Custom JS Function" which outputs "json". 

However, the system ignores the input from the "Format Prompt Values" with the Custom JS Function. 

![examp-1](https://github.com/user-attachments/assets/a5d7da38-2fb1-487b-810e-4232b7ffb5e7)
Has anyone encountered this issue? Is there another solution to more easily distribute prompt variables from the Workers?

**Comment by mingnz** [comment]
Having this issue too, anyone found a solution?

**Comment by 0561joni** [comment]
When you click on Format Prompt Values (from the Prompt Template) you should see something like:
{1 item
text:""
}
when you click on the edit symbol, you can choose the variable from you custom javascript function.
This should solve the issue
